app/experimenter/experiments/admin.py

```diff
@@ -309,7 +310,15 @@ class NimbusExperimentAdmin(
         "channel",
         "firefox_min_version",
     )
-    list_filter = ("status", "publish_status", "application")
+    list_filter = (
+        "status",
+        "publish_status",
+        "is_rollout",
+        "application",
+        "channel",
+        "feature_configs",
+    )
+    search_fields = ("name", "slug")
```

Comment:
I think it would be worth including features too in the search

Reply:
Agreed 👍

Reply:
Oh that's not super practical since it's many related models deep to get to their fields.  I added features on the right filters for filtering by feature?

Reply:
yup that works